packages/docs/scripts/bump-docs.js

```diff
@@ -65,6 +66,27 @@ function genCliDocs (packagesDir) {
   return { 'CLI REFERENCE': commands }
 }
 
+function genVouchDocs(packagesDir) {
+  const docsDir = path.resolve(packagesDir, 'docs', 'docs')
+  const builtDocs = path.resolve(docsDir, 'docs')
+  const vouchDir = path.resolve(packagesDir, 'vouching')
+  const vouchContractsDir = path.resolve(vouchDir, 'contracts')
+  const libDir = path.resolve(vouchDir, 'node_modules', 'zos-lib')
+  const tplDir = path.resolve(vouchDir, 'node_modules', 'tpl-contracts-zos')
+  const ozeDir = path.resolve(vouchDir, 'node_modules', 'openzeppelin-zos')
+  const sidebar =  path.resolve(docsDir, 'website', 'sidebars.json')
+
+  cd(vouchDir)
+  exec('npm install > "/dev/null" 2>&1')
+  exec(`SOLC_ARGS='openzeppelin-zos=${ozeDir},zos-lib=${libDir},tpl-contracts-zos=${tplDir}' npx solidity-docgen ${vouchDir} ${vouchContractsDir} ${docsDir} --exclude mocks`)
```

Comment:
Are we sure that `solidity-docgen` supports multiple external libraries?

Reply:
I think so, but I'm pretty sure they have to be space separated instead of comma separated. So thx!